### DISCUSSION TITLE
Inquiry About Future Licensing Plans for PrimeNG

### DISCUSSION THREAD

**Discussion by rahul-scriptus** [primefaces/community#2694] [PrimeNG]
Dear PrimeNG Support Team,

I am writing to inquire about PrimeNG’s future licensing plans. Specifically, **I would like to know if there are any plans to transition entirely to a premium-only model for future releases of PrimeNG.**

Understanding your future direction on this matter will greatly assist in our planning and decision-making process. Any information you could provide would be greatly appreciated.

Thank you for your time and assistance.

Best regards,
Rahul-scriptus
#PrimeNG

**Comment by qwadrox** [comment]
Hi @rahul-scriptus The core releases of PrimeNG are open-source and free to use. I'm not in the PrimeTek team, but I don't think it is ever going to be closed source and "premium-only" completely. Most UI libraries are open-source and free to use under MIT license. It grants permission for you to use even for commercial applications.

We only need to pay for the long-term updates which provides extended maintenance: https://primeng.org/lts
But this is optional. Mostly enterprise companies benefit from this.
